kie-drools-wb/kie-drools-wb-distribution-wars-cdi1.0/pom.xml

```diff
@@ -343,6 +343,11 @@
       <groupId>org.jbpm</groupId>
       <artifactId>jbpm-services-ejb-timer</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.woodstox</groupId>
+      <artifactId>stax2-api</artifactId>
+      <version>3.1.1</version>
+    </dependency>
```

Comment:
@psiroky as far as i know this is the only way to have multiple versions of a dependency for a single release execution (use of profiles would be an alternative, but it would need 2 different executions).
And enforcement are disable to be able to have such versioning.